luwak/src/test/java/uk/co/flax/luwak/TestInputDocument.java

```diff
@@ -29,6 +30,17 @@
     public ExpectedException expected = ExpectedException.none();
 
     @Test
+    public void testSetDefaultAnalyzer() {
+
+        InputDocument doc = InputDocument.builder("id")
+                    .setDefaultAnalyzer(new StandardAnalyzer())
+                    .addField("test", "value", new StandardAnalyzer()).build();
```

Comment:
Let me go back and check what this was supposed to be doing, maybe it was just testing the setDefaultAnalyzer() method but I'll check the coverage before and after and see?